lstchain/reco/dl1_to_dl2.py

```diff
@@ -436,32 +450,46 @@ def apply_models(dl1, classifier, reg_energy, reg_disp_vector, custom_config={})
     dl2['reco_alt'] = src_pos_reco.alt.rad
     dl2['reco_az'] = src_pos_reco.az.rad
 
-    dl2['reco_type'] = classifier.predict(dl2[classification_features]).astype(int)
-    probs = classifier.predict_proba(dl2[classification_features])[0:, 0]
+    dl2['reco_type'] = classifier.predict(rf_data[classification_features]).astype(int)
+    probs = classifier.predict_proba(rf_data[classification_features])[0:, 0]
     dl2['gammaness'] = probs
     return dl2
 
 
-def set_source_dependent_parameters(data):
+def set_source_dependent_parameters(data, src_pos_camera=[0.195, 0.]):
```

Comment:
I would not set a default value to `src_pos_camera` here, the value should be explicitly and all the time passed.